core/cache/cachetest/eventmatchers.go

```diff
@@ -0,0 +1,63 @@
+package cachetest
+
+import "github.com/juju/juju/core/cache"
+
+var ModelEvents = func(change interface{}) bool {
```

Comment:
why `var ModelEvents = ` construction (even though it's a copy and paste), should we not prevent overriding of these matchers from the sake of reasoning?

Reply:
Yep; done.